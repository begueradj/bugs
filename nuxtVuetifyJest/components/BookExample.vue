<template>
  <v-container
    grid-list-md
    text-xs-center
    fill-height
  >
    <v-layout
      row
      wrap
      align-center
    >
      <v-flex
        xs12
      >
        <h4> {{ title }}</h4>
      </v-flex>
    </v-layout>
  </v-container>
</template>

<script>
export default {
  name: 'BookExample',
  data() {
    return {
      title: 'Mastering Metasploit'
    }
  }
}
</script>
