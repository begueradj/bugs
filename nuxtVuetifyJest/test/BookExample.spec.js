import { mount } from '@vue/test-utils'
import { shallowMount } from '@vue/test-utils'
import { createLocalVue } from '@vue/test-utils'
import Vue from 'vue'
import Vuetify from 'vuetify'
import BookExample from '@/components/BookExample.vue'

const localVue = createLocalVue()
localVue.use(Vuetify)

describe('BookExample.vue', () => {
  test('is a Vue instance', () => {
    const wrapper = shallowMount(BookExample, {
      localVue
    })
    expect(wrapper.html()).toMatchSnapshot()
    expect(wrapper.isVueInstance()).toBeTruthy()
  })
})
