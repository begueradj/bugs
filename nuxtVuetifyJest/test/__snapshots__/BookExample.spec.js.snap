// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookExample.vue is a Vue instance 1`] = `
<v-container-stub tag="div" grid-list-md="" text-xs-center="" fill-height="">
  <v-layout-stub tag="div" row="" wrap="" align-center="">
    <v-flex-stub tag="div" xs12="">
      <h4> Mastering Metasploit</h4>
    </v-flex-stub>
  </v-layout-stub>
</v-container-stub>
`;
